packages/ts-transformers/src/tests/idiomatic-decorators-test.ts

```diff
@@ -1508,6 +1508,7 @@ const tests = (test: uvu.Test<uvu.Context>, options: ts.CompilerOptions) => {
         console.log('click');
       }
     }
+    export { MyElement };
```

Comment:
This seems unfortunate... it's not necessarily idiomatic to split the class declaration and export like this. Is there no other option with 5.1?

Reply:
Ah, mentally I was in the world of the analyzers where we're running the same tests with many versions of TSC. TSC 5.1 does the more idiomatic thing here in the code it emits, and so I've just updated the goldens.